policy/overview-policy-build-time-dependency-vulns.md

```diff
@@ -0,0 +1,98 @@
+# Handling build-time dependency vulnerabilities
+
+## Problem statement
+
+Building software comes with a need for handling and managing dependencies,
+including the management of fallout from any vulnerabilities found within those
+dependencies. This problem is only made worse by large dependency trees that are
+common in many software ecosystems. In this regard, there are a number of issues
+to tackle, with the space of solutions for them being intertwined:
+
+1. As a developer of a library, how do you advertise the discovery of a
+   vulnerability to your dependents?
+2. As a consumer of other libraries, how can you tell if one of your
+   dependencies is affected by a vulnerability?
+3. As a consumer of other libraries, how do you mitigate the effect of a
+   vulnerability in a dependency? Does this change with how far up the
+   dependency tree the issue is? How do you advertise the discovery of such
+   vulnerabilities?
+
```

Comment:
Would it make sense to also make recommendations on "As a developer of a library, how to recommend users report new vulnerabilities in your library"?

Usually, a dedicated "reporting" mechanism (a security emailing list, GitHub's integrated tools to securely report a vulnerability, etc.) should be used vs. a user opening an issue or making some public declaration of their new finding.

Reply:
Added! Let me know if it roughly aligns with what you'd expect.